wercker.yml

```diff
@@ -1,29 +1,13 @@
 box: wercker/default
+
 build:
     steps:
         - shellcheck
 
-        - install-packages:
-            packages: curl
-
-        - script:
-            name: config
-            code: |
-                export KUBERNETES_VERSION="1.15.12"
-                export KUBERNETES_SHA512="316dcb768c0963259f685a4c433150dca3191379a8f2789f39ff50c47b995f702f6f9de89f54a1a260e9ca9fa19dd020ccf7a0a8435e31fe3a7bc5f81b23a643"
-                echo "Installing version $KUBERNETES_VERSION of kubernetes"
-
-        - script:
-            name: fetch kubernetes archive
-            code: |
-                curl -L https://dl.k8s.io/v${KUBERNETES_VERSION}/kubernetes-client-linux-amd64.tar.gz > kubernetes-client-linux-amd64.tar.gz
-                sha512sum kubernetes-client-linux-amd64.tar.gz | grep -q "$KUBERNETES_SHA512"
-                tar xvzf kubernetes-client-linux-amd64.tar.gz
-                kubernetes/client/bin/kubectl version --client
-
```

Comment:
Now, the dependencies are pulled when the step is executed rather than when the step is built.

The reason for this is that Wercker forces a size limit of 50mb, which is insufficient for storing the `gcloud` utility.